core/src/test/java/mediumtest/issues/ShowIssueMediumTests.java

```diff
@@ -0,0 +1,184 @@
+      .contains(1,0,3,4);
+  }
+
+  @Test
+  void it_should_assist_creating_the_binding_if_scope_not_bound() throws Exception{
+    var fakeClient = newFakeClient().assistingConnectingAndBindingToSonarQube("scopeId", CONNECTION_ID, serverWithIssue.baseUrl(), "projectKey").build();
+    backend = newBackend()
+      .withSonarQubeConnection(CONNECTION_ID, serverWithIssue)
+      .withUnboundConfigScope("scopeId")
+      .withEmbeddedServer()
+      .build(fakeClient);
+
+    HttpRequest request = openIssueWithProjectAndKeyRequest("&issue=" + ISSUE_KEY, "&project=" + PROJECT_KEY);
+    var response = java.net.http.HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
+
+    assertThat(response.statusCode()).isEqualTo(200);
+    assertThat(fakeClient.getMessagesToShow()).isEmpty();
+    await().atMost(2, TimeUnit.SECONDS).untilAsserted(() -> assertThat(fakeClient.getIssueParamsToShowByIssueKey()).containsOnlyKeys(ISSUE_KEY));
+    assertThat(fakeClient.getIssueParamsToShowByIssueKey().get(ISSUE_KEY).getRuleKey()).isEqualTo(RULE_KEY);
+  }
+
+  @Test
+  void it_should_assist_creating_the_connection_when_server_url_unknown() throws Exception {
+    var fakeClient = newFakeClient().assistingConnectingAndBindingToSonarQube("scopeId", CONNECTION_ID, serverWithIssue.baseUrl(), "projectKey").build();
+    backend = newBackend()
+      .withUnboundConfigScope("scopeId")
+      .withEmbeddedServer()
+      .build(fakeClient);
+
+    HttpRequest request = openIssueWithProjectAndKeyRequest("&issue=" + ISSUE_KEY, "&project=" + PROJECT_KEY);
+    var response = java.net.http.HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
+
+    assertThat(response.statusCode()).isEqualTo(200);
+    assertThat(fakeClient.getMessagesToShow()).isEmpty();
+    await().atMost(2, TimeUnit.SECONDS).untilAsserted(() -> assertThat(fakeClient.getIssueParamsToShowByIssueKey()).containsOnlyKeys(ISSUE_KEY));
+    assertThat(fakeClient.getIssueParamsToShowByIssueKey().get(ISSUE_KEY).getRuleKey()).isEqualTo(RULE_KEY);
+  }
+
+  @Test
+  void it_should_fail_request_when_issue_parameter_missing() throws Exception{
+    backend = newBackend()
+      .withEmbeddedServer()
+      .build();
+
+    HttpRequest request = openIssueWithProjectAndKeyRequest("&issue=issueKey", "");
+    var response = java.net.http.HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
+    assertThat(response.statusCode()).isEqualTo(400);
+  }
+
+  @Test
+  void it_should_fail_request_when_project_parameter_missing() throws Exception{
+    backend = newBackend()
+      .withEmbeddedServer()
+      .build();
+
+    HttpRequest request = openIssueWithProjectAndKeyRequest("", "&project=projectKey");
+    var response = java.net.http.HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
+    assertThat(response.statusCode()).isEqualTo(400);
```

Comment:
What we also tried to do with style guidelines is to let empty spaces between the Given, When and Then sections, so I would suggest adding an empty line before assertions